Makefile

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright (c) 2014 Rick Salevsky <rsalevsky@suse.de>
+# Copyright (c) 2014, 2015, 2016 Karl Eichwalder <ke@suse.de>
+# Copyright (c) 2015, 2016 Stefan Knorr <sknorr@suse.de>
+#
+
+.PHONY: clean pdf text single-html yast-html validate
+
+DAPS_COMMAND=daps
+
+XSLTPROC_COMMAND = xsltproc \
+--stringparam generate.toc "/article toc" \
+--stringparam generate.section.toc.level 0 \
+--stringparam section.autolabel 1 \
+--stringparam section.label.includes.component.label 2 \
+--stringparam variablelist.as.blocks 1 \
+--stringparam toc.section.depth 2 \
+--stringparam toc.max.depth 3 \
+--stringparam show.comments 0 \
+--stringparam profile.os "$(PROFOS)" \
+--xinclude --nonet
+
+prereqs = DC-release-notes adoc/MAIN.release-notes.adoc
+daps_xslt_rn_dir = /usr/share/daps/daps-xslt/relnotes
+
+text_params =
+
+webaccessparams = --stringparam="homepage=https://www.suse.com" \
+    --stringparam="overview-page=https://www.suse.com/releasenotes" \
+    --stringparam="overview-page-title=Back\ to\ Release\ Notes\ for\ SUSE\ products"
+ifdef NOWEBACCESS
+  webaccessparams = --stringparam="build.for.web=0"
+endif
+
+ifdef NOBUGLINK
+  buglink = --stringparam="use.tracker.meta=0"
+endif
+
+YAST_PROFILED_FILE := build/.profiled/noprofile/release-notes.xml
+
+all: single-html yast-html pdf text
+
+validate: $(prereqs)
+	$(DAPS_COMMAND) -d $< validate
+
+pdf: build/release-notes/release-notes_color_en.pdf
+build/release-notes/release-notes_color_en.pdf: $(prereqs)
+	$(DAPS_COMMAND) -vv -d $< pdf
+
+single-html: build/release-notes/single-html/release-notes/index.html
+build/release-notes/single-html/release-notes/index.html: $(prereqs)
+	$(DAPS_COMMAND) -d $< html --single \
+	  --param="toc.section.depth=2" \
+	  $(webaccessparams) \
+	  $(buglink)
+
+
+yast-html: build/release-notes/yast-html/release-notes.html
+build/release-notes/yast-html/release-notes.html: $(YAST_PROFILED_FILE)
+	mkdir -p build/release-notes/yast-html/; \
+	$(XSLTPROC_COMMAND) $(daps_xslt_rn_dir)/yast.xsl $< > $@
```

Comment:
Do we need that. CAP can't be installed with YaST, so we would never show its release notes in YaST.